tests/MongoDB.Driver.Core.Tests/Core/Operations/CreateIndexesOperationTests.cs

```diff
@@ -274,7 +236,6 @@ public void constructor_should_throw_when_messageEncoderSettings_is_null()
         public void Execute_should_send_session_id_when_supported(
             [Values(false, true)] bool async)
         {
-            RequireServer.Check().Supports(Feature.CreateIndexesCommand);
```

Comment:
Keep the RequireServer.Check() part:

    RequireServer.Check();

Reply:
Done for all removed `RequireServer.Check();` occurrences.